src/state/models/media/gallery.ts

```diff
@@ -100,10 +103,10 @@ export class GalleryModel {
   async addFromUris(uris: InitialImageUri[]) {
     for (const uriObj of uris) {
       this.add({
-        mime: 'image/jpeg',
```

Comment:
This field was being silently ignored by the constructor to `ImageModel`, so I removed it here when I changed the constructor's options type